localstack/services/awslambda/lambda_utils.py

```diff
@@ -429,3 +429,12 @@ def get_lambda_event_filters_for_arn(lambda_arn: str, event_arn: str) -> List[Di
     ]
 
     return event_filter_criterias
+
+
+def has_data_filter_criteria(filters: List[FilterCriteria]) -> bool:
+    for filter in filters:
+        for rule in filter["Filters"]:
```

Comment:
```suggestion
        for rule in filter.get("Filters", []):
```

for additional safety